utils/frame/frame-utilities-cli/src/transfer.rs

```diff
@@ -0,0 +1,171 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Implementation of the `transfer` cli subcommand for nodes that use the pallet-balances crate.
+
+use sc_cli::{
+    Error, SharedParams, pair_from_suri, with_crypto_scheme,
+    CryptoSchemeFlag, decode_hex, CliConfiguration, KeystoreParams,
+};
+use structopt::StructOpt;
+use std::{str::FromStr, fmt::Display};
+use codec::Encode;
+use sp_runtime::MultiSigner;
+use std::convert::TryFrom;
+use sp_core::crypto::Ss58Codec;
+use frame_utils::{
+    AddressFor, IndexFor, BalanceFor, BalancesCall, AccountIdFor, SignedExtensionProvider,
+};
+use crate::utils::create_extrinsic_for;
+
+/// The `transfer` command
+#[derive(Debug, StructOpt, Clone)]
+#[structopt(
+    name = "transfer",
+    about = "Author and sign a Node pallet_balances::Transfer transaction with a given (secret) key"
+)]
+pub struct TransferCmd {
+    /// The number of units to transfer.
+    #[structopt(long)]
+    amount: String,
+
+    /// The signing secret key URI.
+    #[structopt(long)]
+    from: String,
+
+    /// The signing account's transaction index.
+    #[structopt(long)]
+    index: String,
```

Comment:
Why is that a `String`?

Take a look at `BlockNumber` in `params/mod.rs` to get an idea how this should be implemented. (`BlockNumber` should be renamed and generalized for this)